typings/resize-observer-polyfill/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// eslint-disable-next-line import/no-default-export
+export default ResizeObserver;
```

Comment:
Maybe we can remove `resize-observer-polyfill` as it's implemented by all the supported browsers? https://caniuse.com/resizeobserver

Reply:
It's mostly removed, but there are some instances where it will need follow-up from the teams. I will create issues for them.